addresses/80001-mumbai-staging.json

```diff
@@ -59,105 +63,125 @@
+      "args": [],
+      "interfaceId": "0x6db2c812"
     },
     {
       "name": "DisputeResolverHandlerFacet",
       "address": "0x826A551366aCB0e8B97a645bc8d61FbB3Cf90ca1",
-      "args": []
+      "args": [],
+      "interfaceId": "0x6db2c812"
     },
     {
       "name": "AgentHandlerFacet",
       "address": "0x954E0018cD1acc808774048231417c5b39687E22",
-      "args": []
+      "args": [],
+      "interfaceId": "0x6db2c812"
     },
     {
       "name": "BundleHandlerFacet",
       "address": "0x75a762D427f584b49Df816C7B8EF35f366C5021e",
-      "args": []
+      "args": [],
+      "interfaceId": "0x7b53dece"
     },
     {
       "name": "DisputeHandlerFacet",
       "address": "0x506040028eB82d9bcD04f0834E7413D580C12777",
-      "args": []
+      "args": [],
+      "interfaceId": "0xd9ea8317"
     },
     {
       "name": "ExchangeHandlerFacet",
       "address": "0xE9745a15ff08f4a1Ef9f35FafF6afc5C3c81d92A",
-      "args": []
+      "args": [],
+      "interfaceId": "0xbc114381"
     },
     {
       "name": "FundsHandlerFacet",
       "address": "0x2330876E961a1A247f38813B78CdD4c00c7f2852",
-      "args": []
+      "args": [],
+      "interfaceId": "0x18834247"
     },
     {
       "name": "GroupHandlerFacet",
       "address": "0x44132DC5471F013530a5Fa69b89B651073Ca093e",
-      "args": []
+      "args": [],
+      "interfaceId": "0xe2bf2256"
     },
     {
       "name": "MetaTransactionsHandlerFacet",
       "address": "0x76C3bf69519ACc8Ad5fF2B720Ef1F623754e1842",
-      "args": []
+      "args": [],
+      "interfaceId": "0xd25fcdc1"
     },
     {
       "name": "OfferHandlerFacet",
       "address": "0x89682edb5A902A49d73Bf3139Eaa9c52F1551b63",
-      "args": []
+      "args": [],
+      "interfaceId": "0xde051f34"
     },
     {
       "name": "OrchestrationHandlerFacet",
       "address": "0x6e599ec84e16B9444bE392eD489493aebB139692",
-      "args": []
+      "args": [],
+      "interfaceId": "0xb698ded2"
     },
     {
       "name": "TwinHandlerFacet",
       "address": "0xa6Ff006d7a3A66AE432381c0030Ea6A7561A0A36",
-      "args": []
+      "args": [],
+      "interfaceId": "0x60b30e70"
     },
     {
       "name": "PauseHandlerFacet",
       "address": "0xd65517cc0c987AfCd1Ae62e281410f8F1fA636dD",
-      "args": []
+      "args": [],
+      "interfaceId": "0x9ddb8ca6"
     },
     {
       "name": "BosonVoucher Logic",
       "address": "0x7Ec228D90F62116c607a556eb0610C056E097581",
-      "args": []
+      "args": [],
+      "interfaceId": ""
     },
     {
       "name": "BosonVoucher Beacon",
       "address": "0xaFca5eAA6a6826a405c60110942150979Ff78281",
       "args": [
         "0xf9719c7e641964D83cC50ea2d4d0D4e6C300d50E",
         "0x7Ec228D90F62116c607a556eb0610C056E097581"
-      ]
+      ],
+      "interfaceId": ""
     },
     {
       "name": "BosonVoucher Proxy",
       "address": "0xC54A578dbcBD57d844976eABDA61B853d0a0a724",
-      "args": []
+      "args": [],
+      "interfaceId": ""
     }
   ]
```

Comment:
Do we not have a way of getting the interface Ids of the voucher contracts, or do we not need them?

Reply:
Technically we could get them, but we don't really need them, since they live outside the diamond and will never be part of diamond upgrade.